pandas/io/excel/_base.py

```diff
@@ -787,7 +787,8 @@ class ExcelFile:
         If a string or path object, expected to be a path to xls or xlsx file.
```

Comment:
The short summary of this docstring still says that it uses `xlrd` - can you update that while touching these?

Reply:
Thank you for pointing it out. Just pushed a change.